### ISSUE TITLE
[Request]: Pickup selector

### ISSUE THREAD

**Issue by kizzoknight** [sanjay900/guitar-configurator#63]
### Please describe your request.

Could you assign the pickup selector potentiometer to an axis for the rock band guitar so we can set as whatever we want in game

### What operating system are you using?

Windows 10

### What microcontroller are you making a request for?

pi pico

### How will the microcontroller be programed for this request?

Direct

**Comment by sanjay900** [comment]
You already can do this, you just need to set your guitar as a rockband guitar not a guitar hero one.

**Comment by kizzoknight** [comment]
yeah its set as rockband guitar and configured in the software and working but doesnt show as an axis in game or in windows

**Comment by sanjay900** [comment]
are you using PS3 mode or xinput

**Comment by kizzoknight** [comment]
xinput

**Comment by kizzoknight** [comment]
Uploading Guitar Configurator 2022-12-21 01-36-56.mp4…

**Comment by kizzoknight** [comment]
https://youtu.be/UXuJ1cqD4ic

This slider moving was just me leaning on the whammy

**Comment by sanjay900** [comment]
swap your ground and vcc wires

**Comment by kizzoknight** [comment]
![image](https://user-images.githubusercontent.com/55956476/208801285-3f2949b1-6ef6-490b-bf31-590074718019.png)
thats not worked im afraid ive tried it with min and max both ways round and inverted and not inverted

**Comment by sanjay900** [comment]
No idea what's going on then you'll just have to wait for the new firmware

**Comment by sanjay900** [comment]
So i read this again, and i think i might actually know what the issue is here
I don't think xinput on windows understands how the slider bar or pickup selector works.

I suspect if you use PS3 mode, the pickup selector will just show up and be usable.
